### ISSUE TITLE
Date format differences when year is included or not

### ISSUE THREAD

**Issue by grrrrr** [m-i-n-a-r/birday#168]
Can it be possible to have Birday have a standard format for the birthday.

Currently, `3.1.1 (21)`, Birday displays the date differently based on whether a year is included or not.

e.g. 

- with year:   day month year
- without year:   month, day.

It would be useful to have both instances display the same,  preferably  dd-mm[-yyyy]  and or [yyyy-]mm-dd

**Comment by m-i-n-a-r** [comment]
I agree. The issue here is that there's no way to obtain a localized day/month format using LocalDate in Kotlin. And this means that i had to choose a specific format for year-less events and use it as a fixed format independently from the locale. This **IS** indeed an issue, but i tried finding a workaround for months without succeeding, so I appreciate any suggestion 😄
